src/components/Kv/KvCauseSelector.vue

```diff
@@ -0,0 +1,250 @@
+<template>
+	<div class="kv-cause-selector">
+		<template v-if="asIcon">
+			<div class="kv-cause-selector__circle">
+				<img
+					class="kv-cause-selector__img"
+					:srcset="`${causeImage2xSrc} 2x`"
+					:src="causeImageSrc"
+					loading="lazy"
+					width="120"
+					height="120"
+					alt=""
+				>
+			</div>
+		</template>
+		<template v-else>
+			<input
+				class="kv-cause-selector__checkbox"
+				:type="asRadio ? 'radio' : 'checkbox'"
+				:name="asRadio ? 'kv-cause-selector' : cause"
+				:value="asRadio ? cause : checked"
+				:checked="checked"
+				:disabled="disabled"
+				:id="`cause-selector-${cause}-${asRadio ? 'radio' : 'checkbox'}`"
+				@change="onChange($event)"
+			>
+			<label
+				class="kv-cause-selector__label"
+				:for="`cause-selector-${cause}-${asRadio ? 'radio' : 'checkbox'}`"
+			>
+				<div class="kv-cause-selector__circle">
+					<div class="kv-cause-selector__check-icon-wrapper">
+						<kv-icon
+							class="kv-cause-selector__check-icon"
+							name="checkmark"
+						/>
+					</div>
+					<img
+						class="kv-cause-selector__img"
+						:srcset="`${causeImage2xSrc} 2x`"
+						:src="causeImageSrc"
+						loading="lazy"
+						width="120"
+						height="120"
+						alt=""
+					>
+				</div>
+				<div class="kv-cause-selector__text">{{ cause }}</div>
+			</label>
+		</template>
+	</div>
+</template>
+
+<script>
+import { paramCase } from 'change-case';
+import KvIcon from '@/components/Kv/KvIcon';
+
+const imageRequire = require.context('@/assets/images/cause-selector/', true);
+
+export default {
+	components: {
+		KvIcon
+	},
+	model: {
```

Comment:
Why do we need this here? I think this only applies if you are using v-model directly

Reply:
Good catch, left over from an earlier iteration.